articles/vmware/vmw-how-install-rhui-vm.md

```diff
@@ -35,31 +35,47 @@ There are two pre-requisites for performing this operation:
 
 ## Installation
 
-You can find the installer RPM files in the UKCloud public catalog under **media**, for example, **UKCloud-RHUI-Standard-Assured**.
+You can find the installer RPM files in the following locations:
 
-If you require the high availability (HA) package, raise a Service Request directly via the [My Calls](https://portal.skyscapecloud.com/support/ivanti) section of the UKCloud Portal and it will be uploaded to your private catalog.
+[IL2 RHEL 6 Standard](https://cas.frn00006.ukcloud.com/Docs/UKCloud_Shared_Services/IL2-Client-RHEL6-Standard-2.1-1.noarch.rpm?AWSAccessKeyId=438-1048-5-aefff7-1&Expires=1625261766&Signature=zCxurIGSgb2TLrr2ISgqngRuTeU%3D)
 
-1. Mount the `UKCloud RHUI-***.iso` to the VM (or place the contents in an accessible location such as NFS share, FTP, and so on).
+[IL2 RHEL 7 Standard](https://cas.frn00006.ukcloud.com/Docs/UKCloud_Shared_Services/IL2-Client-RHEL7-Standard-2.1-1.noarch.rpm?AWSAccessKeyId=438-1048-5-aefff7-1&Expires=1625261781&Signature=txJ05hPU56lncP%2Bj8hvdgfJJFok%3D)
```

Comment:
We shouldn't be exposing the terms IL2 and IL3 to customers anymore (it's old terminology). Use Assured and Elevated instead

Reply:
Thanks - Corrected